[Improvement-4971][alert|dao|service]Remove redundant PropertyUtils.class

Ts686 opened the pull request:
## *Tips*
- *Thanks very much for contributing to Apache DolphinScheduler.*
- *Please review https://dolphinscheduler.apache.org/en-us/community/development/pull-request.html before opening a pull request.*

this closes #4971 

## What is the purpose of the pull request

*(For example: This pull request adds checkstyle plugin.)*

## Brief change log

*(for example:)*
  - *Add maven-checkstyle-plugin to root pom.xml*

## Verify this pull request

*(Please pick either of the following options)*

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

*(example:)*

  - *Added dolphinscheduler-dao tests for end-to-end.*
  - *Added CronUtilsTest to verify the change.*
  - *Manually verified the change by testing locally.*

Ts686 committed:
[Improvement][alert|dao|service]
1. Remove redundant PropertyUtils.class (#4971)
2. Remove useless import package.

Ts686 committed:
[Improvement][alert|dao|service]
1. Remove redundant PropertyUtils.class (#4971)
2. Remove useless import package.

CalvinKirs commented:
I understand should be inited when the respective application starts

Ts686 commented:
> I understand should be inited when the respective application starts

Now,this commit is the common.properties file loaded in the static code module , the alert.properties file loaded when the Alert Server started. 

u mean just load the datasource.properties file when the master sever, worker server and api server started.

Is it right?

CalvinKirs commented:
> > I understand should be inited when the respective application starts
> 
> Now,this commit is the common.properties file loaded in the static code module , the alert.properties file loaded when the Alert Server started.
> 
> u mean just load the datasource.properties file when the master sever, worker server and api server started.
> 
> Is it right?

I think it should be like this

CalvinKirs commented:
However, you can do this too.

Ts686 committed:
[Improvement][alert|dao|service]
1. Remove redundant PropertyUtils.class (#4971)
2. Remove useless import package.

Ts686 committed:
Merge branch 'dev' of github.com:apache/incubator-dolphinscheduler into remove-redundant

CalvinKirs commented on dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/PropertyUtils.java:
```diff
@@ -51,7 +49,13 @@ private PropertyUtils() {
+    /**
+     * init properties
+     */
+    public static void loadPropertyFile(String... propertyFiles) {
         for (String fileName : propertyFiles) {
             InputStream fis = null;
             try {
```
There may be concurrency issues

Ts686 replied:
> There may be concurrency issues

u mean synchronized method will be better？

Ts686 replied:
but this method invoked only through static code module or the application start  , I don't think it has concurrency issues，right ?

CalvinKirs replied:
Your own rightness should not be tied to the specific business.

CalvinKirs replied:
Your own rightness should not be tied to the specific business.

CalvinKirs commented:
You don't seem to have modified it?

Ts686 commented:
> You don't seem to have modified it?

Yes, I'm still working.I'll modify it when I get home. haha  
The company has no external network.

Ts686 committed:
Merge branch 'dev' of github.com:apache/incubator-dolphinscheduler into remove-redundant

Ts686 committed:
[Improvement][alert|dao|service]
1. Remove redundant PropertyUtils.class (#4971)
2. Remove useless import package.

CalvinKirs reviewed (approved):
LGTM

CalvinKirs commented:
The way the master and worker read configuration files may also be changed to this way, gradually removing Spring's related dependencies.

lgcareer reviewed (approved):
+1
